catboost/tutorials/apply_model/coreml_export_tutorial.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# CatBoost and CoreML tutorial — Iris dataset"
```

Comment:
Let us drop Name and Ticket features as they can't be one-hot features (currently, one_hot_max_size can not be more than 255).

Let's remove this comment, since
1. It is not true starting from previous release
2. It doesn't make sence to one-hot these features, since there's a single object with this value, so there will be overfitting.